### ISSUE TITLE
llvm preset missing `LLVMConstGEP2` and `LLVMConstInBoundsGEP2`

### ISSUE THREAD

**Issue by heshanpadmasiri** [bytedeco/javacpp-presets#1264]
It seems [`LLVMConstGEP2`](https://llvm.org/doxygen/group__LLVMCCoreValueConstantExpressions.html#gae355c965dacae2f840ad9c1e8b60f885) and [`LLVMConstInBoundsGEP2`](https://llvm.org/doxygen/group__LLVMCCoreValueConstantExpressions.html#ga7bd51db0c1eec33d38b10cccc356d35f) functions are missing in the llvm presets for llvm-15
Affected version: 15.0.3-1.5.8

**Comment by saudet** [comment]
That's right, those functions have never been implemented, see https://github.com/llvm/llvm-project/issues/40707.

**Comment by matsjla** [comment]
Unsure how this API is going to work with LLVM's new opaque pointers anyway, probably best to wait and see what happens to the LLVM C API in the future

**Comment by heshanpadmasiri** [comment]
@matsjla it seems that LLVM did add implementations of `LLVMConstGEP2` and `LLVMConstInBoundsGEP2` with [this](https://reviews.llvm.org/D115787) and I managed to confirm that they work with llvm 15's C-API. I am not entirely clear as to why the original issue is still open on the llvm repo.

**Comment by saudet** [comment]
If that's the case, please try to remove their "skip()" from the presets and open a pull request! Thanks

**Comment by saudet** [comment]
Done in commit https://github.com/bytedeco/javacpp-presets/commit/bfb993988291d3a744c520edbd1163e56ec1ab2e. Please give it a try with the snapshots: http://bytedeco.org/builds/
